playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -256,6 +256,99 @@ def get_debugger_warnings(data):
         return None
     return message
 
+@action
+def debugger_stack_trace(event: PodEvent, params: DebuggerParams):
+    """
+    Prints a stack track of a python process and child threads
+
+    Create a finding with the stack trace results.
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.info(f"debugger_stack_trace - pod not found for event: {event}")
+        return
+
+    process_finder = ProcessFinder(pod, params, ProcessType.PYTHON)
+    if not params.pid and not process_finder.get_lowest_relevant_pid():
+        logging.info(f"debugger_stack_trace - no relevant pids")
+
+    # if params pid is set, this will be returned, if not we return the parent process
+    pid = process_finder.get_lowest_relevant_pid()
+    finding = Finding(
+        title=f"Stacktrace on pid {pid}:",
+        source=FindingSource.MANUAL,
+        aggregation_key="debugger_stack_trace",
+        subject=FindingSubject(
+            pod.metadata.name,
+            FindingSubjectType.TYPE_POD,
+            pod.metadata.namespace,
+        ),
+    )
+    event.add_finding(finding)
+    cmd = f"debug-toolkit stack-trace {pid}"
+    output = RobustaPod.exec_in_debugger_pod(
+            pod.metadata.name,
+            pod.spec.nodeName,
+            cmd,
+        )
+    blocks =[]
+    for thread_output in output.split('\n\n'):
+        if thread_output.startswith('Current thread'):
+            # this is the thread we are getting the stack trace from, not relevant for debugging
+            continue
+        if thread_output:
+            blocks.append(MarkdownBlock(f"```\n{thread_output}\n```"))
+    finding.add_enrichment(blocks, annotations={SlackAnnotations.ATTACHMENT: True})
+
+
+@action
+def advanced_debugging_options(event: PodEvent, params: DebuggerParams):
+    """
+
+    Create a finding with alternative debugging options for received processes ; i.e. Stack-trace or Memory-trace.
+
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.info(f"advanced_debugging_options - pod not found for event: {event}")
+        return
+    finding = Finding(
+        title=f"Advanced debugging for pod {pod.metadata.name} in namespace {pod.metadata.namespace}:",
+        source=FindingSource.MANUAL,
+        aggregation_key="advanced_debugging_options",
+        subject=FindingSubject(
+            pod.metadata.name,
+            FindingSubjectType.TYPE_POD,
+            pod.metadata.namespace,
+        ),
+    )
+    event.add_finding(finding)
+    finding.add_enrichment(
+        [MarkdownBlock(f"Please select an advanced debugging choice:")]
+    )
+    process_finder = ProcessFinder(pod, params, ProcessType.PYTHON)
+    relevant_processes_pids = process_finder.get_pids()
+    if not relevant_processes_pids :
+        logging.info(f"No processes selected for advanced_debugging_options")
```

Comment:
What would that look like in slack?
I think it will be: "Please select an advanced debugging choice:", with nothing to select, cause we already created the finding and enrichment.

Can we make it more user friendly ?